src/index.scss

```diff
@@ -347,9 +347,17 @@ header {
     .nav-item:not(:last-child){
       .nav-link {
           border-right: 0;
+
+          @media screen and (max-width: 567px) {
+            border-right: solid 1px #e9e6e4;
+          }
       }
     }
   }
+
+  button {
+    font-family: inherit;
+  }
```

Comment:
Once the theme is applied, the font-family issue will be resolved. so we don't need to add this css.
`npm install --no-save "@edx/brand@git+https://[git@github.com](mailto:git@github.com)/edx/brand-edx.org#master"`

Reply:
@sundasnoreen12 This theme is related to edx.

Reply:
@vladislavkeblysh Font-family issue has been resolved in master branch so we do not need this change.

Reply:
Fixed